includes/Modules/Sign_In_With_Google.php

```diff
@@ -366,10 +365,9 @@ private function render_signinwithgoogle_woocommerce() {
 		 * Only render the button in a WooCommerce login page if:
 		 *
 		 * - the Sign in with Google module is connected
-		 * - WooCommerce is active
```

Comment:
This WC-specific method is hooked to a WC-specific action, so this active check is redundant.

Reply:
Good catch.

Reply:
Note: as a result, I've removed the respective tests that exercised this condition as it's no longer relevant.